tests/Unit/Application/Regulation/Command/Period/SavePeriodCommandHandlerTest.php

```diff
@@ -63,7 +63,7 @@ public function testCreate(): void
         $command->measure = $measure;
         $command->startTime = $start;
         $command->endTime = $end;
-        $command->applicableDays = [ApplicableDayEnum::MONDAY->value, ApplicableDayEnum::TUESDAY->value];
+        $command->applicableDays = [ApplicableDayEnum::TUESDAY->value, ApplicableDayEnum::MONDAY->value, ApplicableDayEnum::WEDNESDAY->value];
```

Comment:
Proposition : ajouter un petit commentaire, car le fait qu'ils ne soient pas dans l'ordre est intentionnel

```suggestion
        $command->applicableDays = [ApplicableDayEnum::TUESDAY->value, ApplicableDayEnum::MONDAY->value, ApplicableDayEnum::WEDNESDAY->value]; // Out of order
```